.github/workflows/ci.yml

```
@@ -148,3 +202,30 @@ jobs:
               LEGO1/omni/src/video/flic.cpp \
               $action_headers \
             --path LEGO1/omni LEGO1/lego/legoomni
+
+  release:
+    name: 'Release'
+    if: ${{ github.event_name == 'release' }}
```

Does this run automatically on master updates?

---

No. I skipped the trigger of `release.yml`. This has to be
```suggestion
    if: ${{ github.event_name == 'push' }}
```